csvcubed/csvcubed/readers/preconfiguredtemplates.py

```diff
@@ -0,0 +1,81 @@
+"""
+Pre-configured JSON Templates
+-----------------------------
+
+Functionality to help augment JSON files with configuration from some pre-configured templates.
+"""
+import logging
+
+from typing import Dict, Any
+from requests.exceptions import JSONDecodeError, HTTPError
+
+from csvcubed.utils.cache import session
+
+TEMPLATE_BASE_URL = "https://raw.githubusercontent.com/GSS-Cogs/csvcubed/main/csvcubed/csvcubed/readers/{}/templates/{}"
+
+def _get_template_file_from_template_lookup(template_value: str, version_module_path: str) -> str:
+    """
+    Given the `from_template` value, look up the template in the git repo
+    """
+    template_lookup_url =  TEMPLATE_BASE_URL.format(version_module_path, "preset_column_config.json")
+    template_lookup_response = session.get(template_lookup_url)
+    logging.debug("The template lookup/index file: %s", template_lookup_url)
+   
+    if not template_lookup_response.ok:
+        raise HTTPError(f'Unable to get from url {template_lookup_url}. Status code: {template_lookup_response.status_code}')
+
+    try:
+        template_lookup = template_lookup_response.json()
+    except JSONDecodeError as e:
+        raise Exception(f"Could not decode response \n{template_lookup_response}\n from {template_lookup_url}") from e
+
+    template_file = template_lookup.get(template_value)
+    if not template_file:
+        raise Exception("Couldn't find template your looking for.")
```

Comment:
Could we add the name of the template the user provided into this error message? It'll make it easier for users to figure out what they did wrong.